lists/ru.csv

```diff
@@ -1,214 +1,289 @@
-http://www.moscowpride.ru,LGBT,LGBT,2014-04-15,citizenlab,
-http://www.narkop.com,ALDR,Alcohol & Drugs,2014-04-15,citizenlab,
-http://www.narkotiki.name,ALDR,Alcohol & Drugs,2014-04-15,citizenlab,
-http://www.national-socialist.tk,HATE,Hate Speech,2014-04-15,citizenlab,
-http://www.ng.ru,NEWS,News Media,2014-04-15,citizenlab,
-http://www.nosochi2014.com/articles/1864-2014-circassian-genocide-olympics.php,HUMR,Human Rights Issues,2014-04-15,citizenlab,
-http://www.nosochi2014.com/index.php,HUMR,Human Rights Issues,2014-04-15,citizenlab,
-http://www.nuruddin-info.blogspot.com,MILX,Terrorism and Militants,2014-04-15,citizenlab,
-http://www.owl.ru,HUMR,Human Rights Issues,2014-04-15,citizenlab,
-http://www.partypoker.net,GMB,Gambling,2014-04-15,citizenlab,
-http://www.porjati.net,MMED,Media sharing,2014-04-15,citizenlab,
-http://www.putin-itogi.ru,NEWS,News Media,2014-04-15,citizenlab,
-http://www.rabidshare.info,HATE,Hate Speech,2014-04-15,citizenlab,
-http://www.radosvet.net,REL,Religion,2014-04-15,citizenlab,
-http://www.resistance88.com,HUMR,Human Rights Issues,2014-04-15,citizenlab,
-http://www.reuters.com/article/2010/03/21/russia-olympics-protest-iduslde62k02s20100321,NEWS,News Media,2014-04-15,citizenlab,
-http://www.rollitup.org,ALDR,Alcohol & Drugs,2014-04-15,citizenlab,
-http://www.russiafreedomfund.org,LGBT,LGBT,2014-04-15,citizenlab,
-http://www.russiancasino.com,GMB,Gambling,2014-04-15,citizenlab,
-http://www.russianopengames.ru,LGBT,LGBT,2014-04-15,citizenlab,
-http://www.samsunggalaxynote2.info,COMM,E-commerce,2014-04-15,citizenlab,
-http://www.seedjah.com,ALDR,Alcohol & Drugs,2014-04-15,citizenlab,
-http://www.shamilonline.org,MILX,Terrorism and Militants,2014-04-15,citizenlab,
-http://www.shram.kiev.ua,SRCH,Search Engines,2014-04-15,citizenlab,
-http://www.sovinform.ru,POLR,Political Criticism,2014-04-15,citizenlab,
-http://www.ssdv.ru,COMM,E-commerce,2014-04-15,citizenlab,
-http://www.svoboda.org,NEWS,News Media,2014-04-15,citizenlab,
-http://www.thiru.com.sg,COMM,E-commerce,2014-04-15,citizenlab,
-http://www.velesova-sloboda.org,CULTR,Culture,2014-04-15,citizenlab,
-http://www.visatovietnam.us,COMM,E-commerce,2014-04-15,citizenlab,
-http://www.weedy.be,ALDR,Alcohol & Drugs,2014-04-15,citizenlab,
-http://www.wikia.com/wikia,HOST,Hosting and Blogging Platforms,2014-04-15,citizenlab,
-http://www.wuor.ru,HUMR,Human Rights Issues,2014-04-15,citizenlab,
-http://www.yabloko.ru,POLR,Political Criticism,2014-04-15,citizenlab,
-http://www.yandex.ru,NEWS,News Media,2014-04-15,citizenlab,
-http://xn--80aaifmgl1achx.xn--p1ai,GMB,Gambling,2014-04-15,citizenlab,
-http://yesche.biz,ALDR,Alcohol & Drugs,2014-04-15,citizenlab,
-http://zabormozga.com,ALDR,Alcohol & Drugs,2014-04-15,citizenlab,
-http://zhurnal.lib.ru,MISC,Miscelaneous content,2014-04-15,citizenlab,
+url,category_code,category_description,date_added,source,notes
+http://2ch.hk,PORN,Pornography,2014-04-15,citizenlab,
```

Comment:
Hmmm, it's strange that this line got no edits, but it sill looks like
```diff
 -http://2ch.hk,PORN,Pornography,2014-04-15,citizenlab, 
 +http://2ch.hk,PORN,Pornography,2014-04-15,citizenlab,
```

Maybe something is wrong with formatting of this patch...

Reply:
That is probably due to the fact that `^M` != `\n` and some people use windows to edit the `.csv` files.